setup.py

```diff
@@ -13,7 +14,22 @@
 source and metadata can be extracted from PyPI or from local filesystem (local file doesn't
 provide that much information though)."""
 
+class build_py(_build_py):
+    def run(self):
+        # Run the normal build process
+        _build_py.run(self)
+        # Build test data
+        from subprocess import call
+        from shutil import copy
+        call(['python3', 'setup.py', 'sdist'],
```

Comment:
I realize I am a bit late here, but should this be `sys.executable` instead of `'python3'`?